internal/store/spicedb/schema_generator/generator.go

```diff
@@ -3,8 +3,8 @@ package schema_generator
 import (
 	"github.com/odpf/shield/internal/schema"
 
-	sdbcore "github.com/authzed/authzed-go/proto/authzed/api/v0"
 	sdbnamespace "github.com/authzed/spicedb/pkg/namespace"
+	sdbcore "github.com/authzed/spicedb/pkg/proto/core/v1"
```

Comment:
why do we need to change this package?

Reply:
due to spicedb upgrade, we need to upgrade proto as well